tests/unit/protobuf/test_protobuf_normalization.py

```diff
@@ -511,10 +516,200 @@
+    dep = Dependency("NestedValue.proto", Subject("nested_value"), Version(1), no_ref_schema)
+    dependencies = {"NestedValue.proto": dep}
+    fully_qualitifed_simple_name_notation = parse_protobuf_schema_definition(
+        schema_definition=PROTO_WITH_SIMPLE_NAMES,
+        references=None,
+        dependencies=dependencies,
+        validate_references=True,
+        normalize=True,
+    )
+    fully_qualitifed_dot_notation = parse_protobuf_schema_definition(
+        schema_definition=PROTO_WITH_FULLY_QUALIFIED_PATHS,
+        references=None,
+        dependencies=dependencies,
+        validate_references=True,
+        normalize=True,
+    )
+    result = CompareResult()
+    normalize(fully_qualitifed_dot_notation).compare(normalize(fully_qualitifed_simple_name_notation), result)
+    assert result.is_compatible()
+    assert (
+        normalize(fully_qualitifed_dot_notation).to_schema() == normalize(fully_qualitifed_simple_name_notation).to_schema()
+    )
+
+
+TRICKY_DEPENDENCY = """\
+syntax = "proto3";
+package org.my.awesome.customer.v1;
+
+message NestedValue {
+    string value = 1;
+}\
+"""
+
+PROTO_WITH_FULLY_QUALIFIED_PATHS_AND_TRICKY_DEPENDENCY = """\
+syntax = "proto3";
+
+package my.awesome.customer.v1;
+
+import "org/my/awesome/customer/v1/nested_value.proto";
+import "my/awesome/customer/v1/nested_value.proto";
+import "google/protobuf/timestamp.proto";
+
+option csharp_namespace = "my.awesome.customer.V1";
+option go_package = "github.com/customer/api/my/awesome/customer/v1;dspv1";
+option java_multiple_files = true;
+option java_outer_classname = "EventValueProto";
+option java_package = "com.my.awesome.customer.v1";
+option objc_class_prefix = "TDD";
+option php_metadata_namespace = "MyAwesomeCustomerV1";
+option php_namespace = "MyAwesomeCustomerV1";
+option ruby_package = "My::Awesome::Customer::V1";
+
+message EventValue {
+  .my.awesome.customer.v1.NestedValue nested_value = 1;
+
+  google.protobuf.Timestamp created_at = 2;
+}
+"""
+
+
+def test_full_path_and_simple_names_are_not_equal_if_simple_name_is_not_unique() -> None:
+    no_ref_schema = ValidatedTypedSchema.parse(SchemaType.PROTOBUF, DEPENDENCY, normalize=True)
+    tricky_no_ref_schema = ValidatedTypedSchema.parse(SchemaType.PROTOBUF, TRICKY_DEPENDENCY, normalize=True)
+    dep = Dependency("NestedValue.proto", Subject("nested_value"), Version(1), no_ref_schema)
+    tricky_dep = Dependency("TrickyNestedValue.proto", Subject("tricky_nested_value"), Version(1), tricky_no_ref_schema)
+    dependencies = {"NestedValue.proto": dep, "TrickyNestedValue.proto": tricky_dep}
+    fully_qualitifed_dot_notation = parse_protobuf_schema_definition(
+        schema_definition=PROTO_WITH_FULLY_QUALIFIED_PATHS_AND_TRICKY_DEPENDENCY,
+        references=None,
+        dependencies=dependencies,
+        validate_references=True,
+        normalize=True,
+    )
+    normalized_schema = normalize(fully_qualitifed_dot_notation).to_schema()
```

Comment:
isn't this normalized already because `normalze=True`?

Reply:
ah yeah, you are right